values.yaml

```diff
@@ -19,6 +19,11 @@ global:
   privateCaCertsAddToHost:
     enabled: false
     hostDirectory: /etc/docker/certs.d
+    certCopier:
+      repository: alpine
+      tag: 3.14
```

Comment:
@danielhoherd should this be 3.14.1?

Reply:
3.14 is a floating tag, and matches whatever the most recent patch version of 3.14 is. we can do a more precise tag if we want.

Reply:
Cool thanks @danielhoherd .. floating tag is fine.